app/models/authority.rb

```diff
@@ -21,6 +21,9 @@ class Authority < ActiveRecord::Base
   has_many :applications
   has_many :councillors
   has_many :comments, through: :applications
+  has_many :councillor_contributions
+  has_many :suggested_councillors, through: :councillor_contributions
```

Comment:
Is this association necessary? If we're not using it we should remove that line.